services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from db.models import MovieSession, Movie, CinemaHall
+from datetime import datetime
+from services.movie import get_movie_by_id
+from services.cinema_hall import get_cinema_halls
+
+from django.db.models import QuerySet
+
+
+def create_movie_session(
+    movie_show_time: datetime, movie_id: int, cinema_hall_id: int
+) -> None:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+    MovieSession.objects.create(
+        show_time=movie_show_time, movie=movie, cinema_hall=cinema_hall
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+    session_id: int,
+    show_time: datetime = None,
+    movie_id: int = None,
+    cinema_hall_id: int = None,
+) -> None:
+    session = get_movie_session_by_id(session_id)
+    if show_time:
+        session.show_time = show_time
+    if movie_id:
+        session.movie = get_movie_by_id(movie_id)
+    if cinema_hall_id:
+        session.cinema_hall = get_cinema_halls().get(id=cinema_hall_id)
```

Comment:
```suggestion
    if show_time is not None:
        session.show_time = show_time
    if movie_id is not None:
        session.movie_id = movie_id
    if cinema_hall_id is not None:
        session.cinema_hall_id = cinema_hall_id
```

Reply:
It leads to error because we can not assign values directly, he only accepts instance of the classes and I can't find how to pass ids directly. I just can't pass tests with those changes